wp-content/themes/ngisweden/functions/shortcodes/deployed_tools_versions.php

```diff
@@ -3,50 +3,54 @@
 // NGI Deployed Tools Versions
 // Renders the file generated on the cluster during deployment,
 // showing which versions of which tools are currently in production.
+
 function ngisweden_deployed_tools_versions_shortcode($atts_raw){
 
     // Fetch the deployment version file
-    $deployed_file_contents = @file_get_contents(get_template_directory().'/cache/deployed_tools.version');
-
-    $deployment_type = '';
-    $deployed_at = '';
-    $tools = [];
+    $locations = array('sthlm'=>'Stockholm node', 'upps'=>'Uppsala node');
+    $output = '';
+    foreach ($locations as $loc_code => $location){
+      $deployed_file_contents = @file_get_contents(get_template_directory().'/cache/deployed_tools.'.$loc_code.'.version');
 
-    foreach(preg_split("/\r\n|\n|\r/", $deployed_file_contents) as $line){
-        if(substr($line, 0, 14) == '-- Deployed at'){
-            $deployed_at = trim(str_replace('--', '', $line));
-        }
-        else if(substr($line, 0, 2) == '--'){
-            $deployment_type = trim(str_replace('--', '', $line));
-        }
-        else {
-            list($tool, $version) = explode(':', $line, 2);
-            if(strlen($tool) > 0){
-                $tools[] = [trim($tool), trim($version)];
-            }
-        }
-    }
+      $deployment_type = '';
+      $deployed_at = '';
+      $tools = [];
 
-    if(count($tools) > 0){
-        $output = '<div class="deployed-tools-versions">';
-        $output .= '<table class="table table-striped table-hover table-sm small">';
-        $output .= '<thead><tr><th>Tool</th><th>Version</th></tr></thead>';
-        $output .= '<tbody>';
-        foreach($tools as $tool){
-            $output .= '<tr><td>'.$tool[0].'</td><td><code>'.$tool[1].'</code></td></tr>';
-        }
-        $output .= '</tbody>';
-        $output .= '</table>';
-        $output .= '<p>Deployment type: '.$deployment_type.'</p>';
-        $output .= '<p>'.$deployed_at.'</p>';
-        $output .= '</div>';
+      foreach(preg_split("/\r\n|\n|\r/", $deployed_file_contents) as $line){
+          if(substr($line, 0, 14) == '-- Deployed at'){
+              $deployed_at = trim(str_replace('--', '', $line));
+          }
+          else if(substr($line, 0, 2) == '--'){
+              $deployment_type = trim(str_replace('--', '', $line));
+          }
+          else {
+              list($tool, $version) = explode(':', $line, 2);
```

Comment:
To avoid the `Undefined notice` warning, you can just hide warnings from the function call by prefixing it with an `@` (assuming that the error is expected / inconsequential):

```suggestion
              list($tool, $version) = @explode(':', $line, 2);
```

Note it might be coming from `list`, not sure which call it is that's triggering the warning.

Warnings shouldn't show up in production, but they're still nice to squash anyway.

Reply:
It was from the list, I've added the `@` to it.